index.ts

```diff
@@ -149,3 +223,44 @@ async function seedDb(seedFile: string, dynamoDB: DynamoDBClient) {
     update.err(error)
   }
 }
+
+async function resetTableStreams(
+  ddbClient: DynamoDBClient,
+  ddbStreamsClient: DynamoDBStreamsClient,
+  arcTableName: string
+) {
+  const db = await _arcFunctions.tables()
+  const tableName = db.name(arcTableName)
+  const table = await ddbClient.send(
+    new DescribeTableCommand({
+      TableName: tableName,
+    })
+  )
+  const stream = await ddbStreamsClient.send(
+    new DescribeStreamCommand({
+      StreamArn: table.Table?.LatestStreamArn,
+    })
+  )
+  const shardArray = stream.StreamDescription?.Shards
+  if (shardArray && table.Table?.LatestStreamArn) {
+    for (const shard of shardArray) {
+      if (shard.ShardId) {
+        const ShardIterator = (
+          await ddbStreamsClient.send(
+            new GetShardIteratorCommand({
+              StreamArn: table.Table.LatestStreamArn,
+              ShardIteratorType: 'LATEST',
+              ShardId: shard.ShardId,
+            })
+          )
+        ).ShardIterator
+
+        if (ShardIterator) {
+          shardMap[arcTableName].push({
+            ShardIterator: ShardIterator,
```

Comment:
```suggestion
            ShardIterator,
```

I'm surprised that Prettier didn't fix that up for you.